### PR TITLE
Fix/metamask prompt modal

### PR DESCRIPTION
**Pull request by Aliiiu**
# Fix for Prompt Modal for Metamask installation Issue

## How should this be tested?

1.  Disconnect the metamask extension
2. Click on the wallet icon
3. Click on the metamask connector

## Notes or observations

Light Theme
<br />
Screen Shot 2023-08-15 at 12 12 55 AM

<br />
Dark Theme
<br />

Screen Shot 2023-08-15 at 12 13 22 AM

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1679

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
image

you don't have to create new files for the icons, just import RiErrorWarningFill from react-icons/ri.

File: src/components/Layout/WalletDrawer/Connectors.tsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ import {
 } from '@/utils/WalletUtils/fetchWalletBalance'
 import { shortenBalance } from '@/utils/textUtils'
 import { env } from '@/env.mjs'
+import MetaMaskConnectionErrorModal from './MetaMaskConnectionErrorModal'
```
Comment (Emmanueldmlr) [inline src/components/Layout/WalletDrawer/Connectors.tsx:34]: I think we shouldn't limit it to metamask only. it works for other connectors when they are also not active. You might want to rename it to be more general.

File: src/components/Layout/WalletDrawer/MetaMaskConnectionErrorModal.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  isOpen = false,
+}: Partial<ModalProps>) => {
+  const { colorMode } = useColorMode()
+
+  return (
+    <Modal onClose={onClose} isOpen={isOpen} isCentered size="2xl">
+      <ModalOverlay />
+      <ModalContent
+        w="min(95vw, 600px)"
+        _dark={{
```
Comment (Emmanueldmlr) [inline src/components/Layout/WalletDrawer/MetaMaskConnectionErrorModal.tsx:28]: check if we have a semantic token for this.

File: src/components/Layout/WalletDrawer/MetaMaskConnectionErrorModal.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+              h="34px"
+              w="34px"
+            />
+            <Text fontSize={'xl'}>Metamask not detected</Text>
+          </HStack>
+          <ModalCloseButton size="md" my="auto" position="unset" ml="auto" />
+        </ModalHeader>
+        <ModalBody>
+          <Box px={5} pb={9} textAlign={'center'}>
+            Metamask was not detected in your browser, You can acquire MetaMask,
```
Comment (Emmanueldmlr) [inline src/components/Layout/WalletDrawer/MetaMaskConnectionErrorModal.tsx:51]: Make this general so as to work for all other connectors.

File: src/components/Layout/WalletDrawer/MetaMaskConnectionErrorModal.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          bg: 'gray.800',
+        }}
+        rounded={'16px'}
+      >
+        <ModalHeader px={3} fontSize="lg" display="flex" alignItems="center">
+          <HStack spacing={2}>
+            <Image
+              alt="Warning"
+              src={
+                colorMode === 'dark'
```
Comment (Emmanueldmlr) [inline src/components/Layout/WalletDrawer/MetaMaskConnectionErrorModal.tsx:38]: You don't need this when you use the Icon from react-icons/ri.

**Review by Emmanueldmlr — CHANGES_REQUESTED** [review]

File: src/components/Layout/WalletDrawer/Connectors.tsx (hunk 1 above)
Comment (Aliiiu) [reply src/components/Layout/WalletDrawer/Connectors.tsx:34]: Alright

File: src/components/Layout/WalletDrawer/MetaMaskConnectionErrorModal.tsx (hunk 4 above)
Comment (Aliiiu) [reply src/components/Layout/WalletDrawer/MetaMaskConnectionErrorModal.tsx:38]: Noted

**Review by Emmanueldmlr — APPROVED** [review]

**Review by Simplypheyie96 — APPROVED** [review]

**Review by Softdev1 — APPROVED** [review]

**Pull request merged**